CHANGELOG.md

```diff
@@ -10,8 +10,73 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## 🚀 Features
 ## 🐛 Fixes
 ## 🛠 Maintenance
-## 📚 Documentation -->
+## 📚 Documentation
+
+## Example section entry format
+
+- **Headline** via [#PR_NUMBER](https://github.com/apollographql/router/pull/PR_NUMBER)
+
+  Description! And a link to a [reference]
+
+  [reference]: http://link
+
+ -->
+
+# [v0.1.0-alpha.5] 2022-02-15
+
+## :sparkles: Features
+
+- **Apollo Studio usage reporting agent and operation-level reporting** ([PR #309](https://github.com/apollographql/router/pulls/309), [PR #420](https://github.com/apollographql/router/pulls/420))
+
+  While there are several levels of Apollo Studio integration, the initial phase of our Apollo Studio reporting focuses on operation-level reporting.
+
+  At a high-level, this will allow Apollo Studio to have visibility into some basic schema details, like graph ID and variant, and per-operation details, including:
+  
+  - Overall operation latency
+  - The number of times the operation is executed
+  - [Client awareness] reporting, which leverages the `apollographql-client-*` headers to give visibility into _which clients are making which operations_.
+
+  This should enable several Apollo Studio features including the _Clients_ and _Checks_ pages as well as the _Checks_ tab on the _Operations_ page.
+  
+  > *Note:* As a current limitation, the _Fields_ page will not have detailed field-based metrics and on the _Operations_ page the _Errors_ tab, the _Traces_ tab and the _Error Percentage_ graph will not receive data.  We recommend configuring the Router's [OpenTelemetry tracing] with your APM provider and using distributed tracing to increase visibility into individual resolver performance.
+
+  Overall, this marks a notable but still incremental progress toward more of the Studio integrations which are laid out in [#66](https://github.com/apollographql/router/issues/66).
+
+  [Client awareness]: https://www.apollographql.com/docs/studio/metrics/client-awareness/
+  [Schema checks]: https://www.apollographql.com/docs/studio/schema-checks/
+  [OpenTelemetry tracing]: https://www.apollographql.com/docs/router/configuration/#tracing
+
+- **Complete GraphQL validation** ([PR #471](https://github.com/apollographql/router/pull/471) via [federation-rs#37](https://github.com/apollographql/federation-rs/pull/37))
+
+  We now apply all of the standard validations which are defined in the `graphql` (JavaScript) implementation's default set of "[specified rules]" during query planning.
+
+  [specified rules]: https://github.com/graphql/graphql-js/blob/95dac43fd4bff037e06adaa7cfb44f497bca94a7/src/validation/specifiedRules.ts#L76-L103
+
+- **Operations can now be made via `GET` requests** ([PR #429](https://github.com/apollographql/router/pull/429))
```

Comment:
This pull request targets `geal-all-along-the-tower-service` which hasn't landed yet. It will land in next release along with the plugin capabilities

Reply:
WHOOPS.  Good catch.  I've reverted it.

At least the `CHANGELOG` is written.   Maybe cherry-pick the revert onto the new main!